api/src/main/java/org/apache/cloudstack/outofbandmanagement/OutOfBandManagementService.java

```diff
@@ -33,6 +33,9 @@
     ConfigKey<Integer> SyncThreadPoolSize = new ConfigKey<Integer>("Advanced", Integer.class, "outofbandmanagement.sync.poolsize", "50",
             "The out of band management background sync thread pool size", true, ConfigKey.Scope.Global);
 
+    ConfigKey<Integer> outOfBandManagementBackgroundTaskExecutionInterval = new ConfigKey<>("Advanced", Integer.class, "outofbandmanagement.background.task.execution.interval", "4",
```

Comment:
```suggestion
    ConfigKey<Integer> OutOfBandManagementBackgroundTaskExecutionInterval = new ConfigKey<>("Advanced", Integer.class, "outofbandmanagement.background.task.execution.interval", "4",
```

Reply:
@DaanHoogland Why this should initiate with a capital letter?

Reply:
@GutoVeronezi it is just the convention for settings throughout the system. No one will die either way.

Reply:
@DaanHoogland Alright, done.

Reply:
no @GutoVeronezi you had my ok from the start ;)

Reply:
sorry, i now see, you had't sorry. You should have.